pkg/vmimage/data/script.sh

```diff
@@ -5,22 +5,18 @@ exec 2>&1
 export HOME=/root
 cd
 
-echo ,+ | sfdisk --force -u S -N 2 /dev/sda || true
-partprobe
-xfs_growfs /dev/sda2
+mkfs.xfs -f /dev/sdc
+mkdir -p /var/lib/libvirt/images/
+mount /dev/sdc /var/lib/libvirt/images
+
+echo "net.ipv4.ip_forward = 1" >> /etc/sysctl.conf
+sysctl -p /etc/sysctl.conf
 
 yum -y update -x WALinuxAgent # updating WALinuxAgent kills this script
-yum -y install git golang libguestfs-tools-c libvirt-daemon-config-network virt-install
-
-rpm --import https://packages.microsoft.com/keys/microsoft.asc
-cat >/etc/yum.repos.d/azure-cli.repo <<'EOF'
-[azure-cli]
-baseurl=https://packages.microsoft.com/yumrepos/azure-cli
-enabled=1
-gpgcheck=1
-gpgkey=https://packages.microsoft.com/keys/microsoft.asc
-EOF
-yum -y install azure-cli
+yum -y install git golang libguestfs-tools-c libvirt-daemon-config-network virt-install libguestfs-xfs libguestfs-tools python36 python36-devel redhat-rpm-config
+alternatives --set python /usr/bin/python3
+
+pip3 install --upgrade --pre azure-cli --extra-index-url https://azurecliprod.blob.core.windows.net/edge --no-cache-dir
```

Comment:
We're switching from an rpm install of azure-cli to a `pip install` directly to the system, which is typically not recommended as it can irreparably break existing rpm Python installs that share dependencies. Are we doing this now because the rpm isn't available for RHEL8?

Reply:
Need to add comment. https://github.com/Azure/azure-cli/issues/8780 is the reason.